ui/js/selectors/wallet.js

```diff
@@ -60,14 +66,14 @@ export const selectTransactionItems = createSelector(
           //value on transaction, amount on outpoint
           //amount is always positive, but should match sign of value
           const amount = parseFloat(
-            item.amount ? (item.value < 0 ? -1 : 1) * item.amount : item.value
+            item.amount ? item.balance_delta : item.value
           );
 
           return {
             txid: txid,
             date: tx.timestamp ? new Date(parseInt(tx.timestamp) * 1000) : null,
             amount: amount,
-            fee: amount < 0 ? -1 * tx.fee / append.length : 0,
+            fee: amount < 0 ? -1 * tx.fee : 0,
```

Comment:
@kauffj why was `tx.fee` divided by `append.length`. Did it serve any particular purpose.

Reply:
@hackrush01 because this is presenting each individual outpoint as a unique transaction to a user, I think it is less confusing to distribute the fee across all of the outpoints. 

Consider a transaction with say, 5 outpoints. If we show the fee on each outpoint without distributing it, a user is decently likely to interpret the transaction fee as 5x as large as it actually was. I think it is better this fee is shown as 1/5 for each outpoint.